Lesson1/T_Ilya/CPP_ADV_Lesson1/task2.cpp

```diff
@@ -0,0 +1,56 @@
+#include "task.h"
+#include "functions.h"
+
+
+
+namespace MySpace{
+int task2() // -----LIST-----
+{
+    int x{10};
+    int y{5};
+    int a{19};
+    int b{4};
+    std::vector<int> vect_1{};
+    vect_1.reserve(5);
+    std::vector<int> vect_2{};
+    vect_2.reserve(4);
+    std::vector<int> vect_3{};
+    std::list<int> list_1;
+    std::list<int> list_2;
+    MFun::power(x,y,vect_1);
+    std::cout << "vector 1" << std::endl;
+    MFun::print_vect_b(vect_1);
+    MFun::power(a,b,vect_2);
+    std::cout << "vector 2" << std::endl;
+    MFun::print_vect_b(vect_2);
+
+    for (MySpace::ull i{0}; i < vect_1.size(); ++i){
```

Comment:
use size_t instead of unsigned long long.

Reply:
mostly